src/uwloc/data/main.py

```diff
@@ -18,34 +20,52 @@ def fail(msg: str) -> None:
 @app.command()
 def initdb(
     dbpath: str,
+    wavs_dir: str,
     max_units: Annotated[
         int, typer.Argument(help="Maximum number of units (rows) to store in the array")
     ] = MAX_UNITS,
     max_hrs: Annotated[int, typer.Argument(help="Maximum amount of hours to store in the array")] = MAX_HOURS,
 ) -> None:
-    typer.echo(f"Initializing TileDB in {dbpath}")
-    create(dbpath, max_units, max_hrs)
+    start_date: datetime = _find_start_date(wavs_dir)
+    typer.echo(f"Initializing TileDB in {dbpath} with start date: {start_date}")
+    create(dbpath, start_date, max_units, max_hrs)
+    import_wavs(dbpath, wavs_dir)
 
 
 @app.command(name="import")
-def import_wavs(wavs_dir: str, dbpath: str) -> None:
+def import_wavs(
+    dbpath: str,
+    wavs_dir: str,
+) -> None:
     typer.echo(f"Uploading WAV files from {wavs_dir} {dbpath}")
     if not os.path.isdir(dbpath):
         fail(f"TileDB '{dbpath}' not found. Run initdb command first.")
 
-    files = glob.iglob(os.path.join(wavs_dir, "**/*.[wW][aA][vV]"), recursive=True)
-    # TODO: Handle multiple recording files from the same device. These need to be sorted
-    # and laid out correctly in the array
+    files = sorted(glob.iglob(os.path.join(wavs_dir, WAV_PATTERN), recursive=True))
+    for f in files:
+        try:
+            (device, timestamp, rate, data) = read_wav(f)
+            if len(device) == 0:
+                typer.echo(f"{f} is missing the Device ID, skipping")
+                continue
+            secs = data.size // rate
+            typer.echo(f"Importing  {f}. Timestamp: {timestamp} Device: {device}, secs: {secs}")
+            write_row(dbpath, device, timestamp, data)
+        except Exception as e:
+            typer.echo(f"Warning: Failed to import {f}. {e}", err=True)
```

Comment:
Would this quit the program or just print out this warning message?

Reply:
Just print, `err=True` just causes it to output to `stderr` instead of `stdio`

Reply:
This doesn't fail the process? Wouldn't you want to fail if it is a failure to import?

Reply:
We may be importing hundreds of files on there may be a wonky one in there on occasion, so I don't want to fail the entire process because of this. I need to discuss with Marcelo how this should be handled.

Reply:
Got it. I've run into this. Probably good to have a log of the files that failed instead of just printing 😄